ansys/dpf/core/elements.py

```diff
@@ -550,41 +580,69 @@ def map_scoping(self, external_scope):
     
     @property
     def has_shell_elements(self) -> bool:
-        """Returns true if at list one element is a 2D element (shell)"""
+        """Whether at least one element is a 2D element (shell).
+        
+        Returns
+        -------
+        bool
+             ``True`` when successful, ``False`` when failed.
```

Comment:
```suggestion
```
Here we're actually evaluating if something has shell elements or not, rather than if it's successful.  Since we've described the return in the short summary, there's no reason to repeat it and we can just remove this.